atomic-agents/atomic_agents/agents/base_agent.py

```diff
@@ -71,7 +73,10 @@ class BaseAgentConfig(BaseModel):
         0,
         description="Temperature for response generation, typically ranging from 0 to 1.",
     )
-
+    max_tokens: Optional[int] = Field(
+        2048,
+        description="Maximum number of token allowed in the response generation, typically ranging from 1024.",
```

Comment:
Let's make this:
`description="Maximum number of tokens allowed in the response generation."`